src/sql/src/plan/func.rs

```diff
@@ -1127,7 +1127,7 @@ fn coerce_args_to_types(
 
 /// Provides shorthand for converting `Vec<ScalarType>` into `Vec<ParamType>`.
 macro_rules! params {
-    (($($p:expr),*)...) => { ParamList::Repeat(vec![$($p.into(),)*]) };
+    (($p:expr)...) => { ParamList::Variadic($p.into())};
```

Comment:
_super_ minor, but the parens around `$p` are probably superfluous now.

Reply:
Glad you pushed on this––I tried to remove them but couldn't get it to work with `expr` or `literal` metavariables...didn't realize `ident` was an option.

Reply:
Ahh, makes sense. Nice!